modules/distribution/src/main/scripts/fluo

```diff
@@ -131,50 +131,50 @@ function setup_service {
 case "$1" in
```

Comment:
I'm not as proficient at shell scripting as I'd like to be so any comments on this file would be greatly appreciated. Now that the same class is being used as the entry point for almost all commands this script looks a little redundant. I feel like there may be ways of condensing it, but I didn't want to attempt to much in this PR.

Reply:
It does look like it could be condensed, but I am not sure of the best approach.  Would be a good follow on issue.